udata_gouvfr/migrations/2018-04-18-migrate-active-users.js

```diff
@@ -0,0 +1,13 @@
+/*
+ * Ensure active users have a confirmed_at date set.
+ */
+
+var count = 0;
+
+db.user.find({active: true, confirmed_at: null}).forEach(function(user) {
```

Comment:
I think you can safely drop the `forEach` and use `update` instead (faster for this, one query instead of one + `nbusers`):
```javascript
const result = db.user.find(
    {active: true, confirmed_at: null},
    {$set: {'confirmed_at': new Date()}},
    {multi: true}
);
print(`${result.nModified} users migrated.');
```